docs/globalnet.md

```diff
@@ -0,0 +1,71 @@
+# Introduction
+
+Submariner is a tool build to connect overlay networks of different Kubernetes clusters. These clusters can be on different public clouds or OnPremises. An important use case for Submariner is to connect disparage independent clusters into a single cohesive multi-cluster.
+ However, one key limitation of submariner's current design is that it doesn't support overlapping CIDRs across these clusters. Each cluster must use a distinct ClusterCIDR that doesn't conflict or overlap with any other cluster that is going to be part of multicluster.
+
+<!-- TODO: Add diagram showing the problem --> 
+
+This is largely problematic coz most actual deployments use the default CIDR for cluster, this end up using the same CIDR. Changing CIDR on existing clusters is a very disruptive process and requires a cluster restart. So we require a means to allow clusters with Overlapping CIDRs connect together with submariner.
```

Comment:
```suggestion
This is largely problematic because most actual deployments use the default CIDRs for a cluster so every cluster ends up using the same CIDRs. Changing CIDRs on existing clusters is a very disruptive process and requires a cluster restart. So submariner needs a way to allow clusters with overlapping CIDRs to connect together..
```